game_runner.rb

```diff
@@ -0,0 +1,19 @@
+class GameRunner
+  def initialize(stdout)
+    @stdout = stdout
+  end
+
+  def run
+    grid_template = <<~GRID
+         __ __ __
+        |  |  |  |
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+        |  |  |  |
+        |__|__|__|
+    GRID
+
+    @stdout.print(grid_template)
```

Comment:
It's a better practices to use attr_accessors vs direct use of instance variables. Why? It's more better to think of everything in OO as a message/method. Whenever you are thinking about data, you likely have a missing object. So use a reader here.